### ISSUE TITLE
feat: make slideshow fully WCAG 2.1 AA compliant

### ISSUE THREAD

**Issue by ramboz** [hlxsites/petplace#88]
We should make sure the block is fully compliant with WCAG 2.1 AA. We can essentially follow an approach like https://github.com/hlxsites/delta/blob/main/us/en/skymiles/blocks/tabs/aria-tabs.js to create a wrapper web component that abstracts the aria properties and default behavior, and make sure to follow the spec from https://www.w3.org/WAI/ARIA/apg/patterns/carousel/ to do most of the decoration.

To validate the compliance, we can run this through:
- https://wave.webaim.org/ or https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh
- https://accessibe.com/accessscan
- https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd

**Comment by mfrisbey** [comment]
After reviewing the spec and the current slideshow, I've identified the following tasks:

[ ] Add next and previous slide buttons.
[ ] Add a start/stop button.
[ ] Verify that slides are hidden correctly when not visible (i.e. they're not just pushed off the screen, but hidden entirely)
[ ] Slide rotation should stop whenever any element in the slideshow receives keyboard focus. Rotation shouldn't restart until explicitly requested by the user.
[ ] Verify that all buttons in the slideshow implement the [button pattern](https://www.w3.org/WAI/ARIA/apg/patterns/button/).
[ ] The start/stop button should be the first tab stop in the control.
[ ] Generally review `aria` requirements as defined in the spec, and as provided in the example.
  [ ] Remove `aria-roledescription` from the slide selection `tabpanel`
  [ ] Each `tab` should have an accessible name consisting of the tab's title (if possible). At the very least, each tab should have a description similar to "Slide 3"
  [ ] `tablist` element needs an `aria-label`
[ ] Generally verify that tabs meet the [tab pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs).
